packages/guilded.js/lib/structures/Message.ts

```diff
@@ -93,7 +93,7 @@ export class Message extends Base<ChatMessagePayload> {
     }
 
     /* Update message content */
-    update(newContent: string): Promise<Message> {
+    update(newContent: RESTPostChannelMessagesBody | string): Promise<Message> {
         return this.client.messages.update(this.channelId, this.id, newContent).then(() => this);
```

Comment:
need to construct an object in case a string is passed

Reply:
Whoops, nevermind. It's actually not necessary here

Reply:
i was gonna say 💀

Reply:
I’m on mobile and not home haha